src/style.css

```diff
@@ -1,3 +1,80 @@
+:root {
+  --main-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.logo img {
+  width: 40px;
+  height: 40px;
+  vertical-align: bottom;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
```

Comment:
it the checklist you have a requirement not to define height of the container, please make links height to define the height of header

Reply:
@damian2403 one last change left: there is requirement not to hardcode the height on header, please fix if and you're good to go